src/components/VerticalTabs/VerticalTabs.reducers.js

```diff
@@ -0,0 +1,12 @@
+export default {
+	onSelect(state = {}, selectedIndex, { props = {} } = {}) {
+		if (props.isDisabled) {
```

Comment:
Yeah this shouldn't be referencing the last arg at all as it is only used by the consumer.